R/python.R

```diff
@@ -278,6 +278,10 @@ renv_python_discover <- function() {
     "~/opt/python",
     file.path(renv_pyenv_root(), "versions")
   )
+  reticulate <- Sys.getenv("RETICULATE_PYTHON")
+  if (length(reticulate)) {
```

Comment:
```suggestion
  if (nzchar(reticulate)) {
```

Since `Sys.getenv()` will return a zero-length string if the associated environment variable doesn't exist.